analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -69,6 +69,7 @@ class Cargo(
         private const val REQUIRED_CARGO_VERSION = "1.0.0"
         private const val SCOPE_NAME_DEPENDENCIES = "dependencies"
         private const val SCOPE_NAME_DEV_DEPENDENCIES = "devDependencies"
```

Comment:
See #1708 we should follow scope naming in Rust lang docs [1][2] and rename "devDependencies" to "dev-dependencies" and  buildDependencies to "build-dependencies".  The scope "devDependencies" name was only introduced as @boxdot based the Rust implementation of the NPM one.


[1] https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#development-dependencies
[2] https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#build-dependencies

Reply:
Done.